README.md

```diff
@@ -243,7 +246,7 @@ slack:           `slack://tokenA/tokenB/tokenC`
 
 rocketchat:      `rocketchat://[username@]rocketchat-host/token[/channel|@recipient]`
 
-teams:           `teams://token-a/token-b/token-c`
+teams:           `teams://tName/token-a/token-b/token-c`
```

Comment:
Will that be clear enough, or should we link to a a reference explaining how to get tName?

Reply:
Added some explanation and a link